### PR TITLE
added: Push release branches to github repository

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: RELEASE-README.md [hunk 1]
```diff
@@ -451,12 +452,22 @@ If everything is perfect (compiles, Jenkins is all blue, sanity checks succeed a
          * Add this change
          * Commit this change.
 
+
+        
+* Push release branches to github repository
+
+    The release branches rX.X.X.Y should be pushed to the github repository (community=kiegroup/... or product=jboss-integration/...), so the branch<br>
+    is available for all futrure steps. People can access it to review if all commits that should be in the release were commited.<br>
```
Comment (ederign) [inline RELEASE-README.md]: typo.

**Comment by mbiarnes** [comment]
@ederign done

File: RELEASE-README.md [hunk 2]
```diff
@@ -451,12 +452,22 @@ If everything is perfect (compiles, Jenkins is all blue, sanity checks succeed a
          * Add this change
          * Commit this change.
 
+
+        
+* Push release branches to github repository
+
+    The release branches rX.X.X.Y should be pushed to the github repository (community=kiegroup/... or product=jboss-integration/...), so the branch<br>
```
Comment (psiroky) [inline RELEASE-README.md]: Remove `<br>`?

**Comment by mbiarnes** [comment]
@psiroky removed the superfluous <br>

**Pull request merged**